src/monodroid/jni/xxhash.hh

```diff
@@ -0,0 +1,242 @@
+#if !defined (__XXHASH_HH)
+#define __XXHASH_HH
+
+//
+// Based on original code at https://github.com/ekpyron/xxhashct
```

Comment:
Sounds like this code was copied from "elsewhere".

Please move it into `src-ThirdParty`, and update `build-tools/xaprepare/xaprepare/Scenarios/Scenario_ThirdPartyNotices.cs` & co. so that the in-tree `ThirdPartyNotices.txt` appropriately mentions this code.

Reply:
This code is not copied from the above repository verbatim, I modified it quite a bit.